screenpy/speech_tools.py

```diff
@@ -44,3 +46,20 @@ class name: replace each capital letter with a space and a lower-case
         description = f"the {describable.__class__.__name__}"
 
     return description
+
+
+@overload
+def tostring(item: str) -> str:
+    ...
+
+
+@overload
+def tostring(item: T) -> T:
+    ...
+
+
+def tostring(item: Union[str, T]) -> Union[str, T]:
+    """help convert objects to a proper format for logging"""
+    if isinstance(item, str):
+        return repr(item)
+    return item
```

Comment:
I'm a bit confused by this. I didn't peek during this PR because i assumed this function was going to be doing something... else than this. What exactly does this do? Calling `repr` on a string seems to just wrap the string with apostrophes.
```python
>>> repr("Hello!")
"'Hello!'"
```

I'm not sure i fully understand the goal here. Notably, this `tostring` function does not do what the `!r`s you've added elsewhere do, which just do `repr` on everything—string or not. If that's what you wanted, we could probably replace all the `tostring`s everywhere with `repr`s?

Reply:
> I apologize if using it differently in tests vs in code caused confusion

But it's not only in test code... https://github.com/ScreenPyHQ/screenpy/pull/98/files#diff-ad02cf191af6b87150c4b57f97e708473cd43d109fb9d1eb22432f1084fb8c79R81-R83

Reply:
Oh! nope, that's not supposed to be there.  Fixing.